frappe/www/app.py

```diff
@@ -44,13 +44,15 @@ def get_context(context):
 
 	include_js = hooks.get("app_include_js", []) + frappe.conf.get("app_include_js", [])
 	include_css = hooks.get("app_include_css", []) + frappe.conf.get("app_include_css", [])
+	include_icons = hooks.get("app_include_icons", []) + frappe.conf.get("app_include_icons", [])
```

Comment:
```suggestion
	include_icons = hooks.get("app_include_icons", [])
```

I don't see a realistic need to include icons from config. Lets limit it to hooks.